invisible_cities/cities/components_test.py

```diff
@@ -465,6 +471,54 @@ def test_read_wrong_pmt_ids(ICDATADIR):
         next(sns_gen)
 
 
+@mark.parametrize( "norm_strat norm_value".split(),
+                  ( (NormStrategy.kr    , None) # None marks the default value
+                  , (NormStrategy.max   , None)
+                  , (NormStrategy.mean  , None)
+                  , (NormStrategy.custom,  1e3)
+                  ))
+@mark.parametrize("apply_temp", (False, True))
+def test_hits_corrector_valid_normalization_options( correction_map_filename
+                                                   , norm_strat
+                                                   , norm_value
+                                                   , apply_temp ):
```

Comment:
A docstring to explain this test may be useful, although I think its self explanatory. I think `test_hits_corrector_invalid_normalization_options_raises` doesn't require one as the comments describing the fixtures + the name of the test explain (at least to me) what the test is looking to do.